lib/src/utils.dart

```diff
@@ -34,9 +35,9 @@ Iterable<File> listDartFilesIn(String dirPath, List<String> excludedDirs) {
 
   return new Directory(dirPath).listSync(recursive: true).where((entity) {
     if (entity is! File) return false;
-    if (entity.path.contains('/packages/')) return false;
-    if (!entity.path.endsWith('.dart')) return false;
-    if (excludedDirs.any(entity.path.startsWith)) return false;
+    if (entity.path.split(p.separator).contains('packages')) return false;
```

Comment:
This is probably fine but in the future I'd recommend using `p.split`, and not `.split(p.separator)`.

For example:
```dart
// Multiple slashes should collapse to one in most cases
print('foo//bar'.split(p.separator)); // [foo, , bar]
print(p.split('foo//bar'));           // [foo, bar]

// In Windows, you can mix slashes
print(r'foo/bar\baz'.split(p.windows.separator)); // [foo/bar, baz]
print(p.windows.split(r'foo/bar\baz'));           // [foo, bar, baz]
```

Paths have so many edge-cases, and it's usually better to not make any assumptions and let the `path` package take care of it for you.